server/src/index.js

```diff
@@ -21,16 +21,21 @@ const dataSources = () => ({
   mainAPI: new MainAPI(store),
 });
 
-const APP_SECRET = process.env.SECRET || '';
+const APP_SECRET = process.env.SECRET || ' ';
```

Comment:
Why are we adding a whitespace for default secret? Whitespaces are difficult to handle & detect at debugging time & prone to error (e.g. string normalization turns that into empty space). Can we use null?

Reply:
Unfortunately, In jwt library need to at least one character for second parameter(APP_SECRET)

Reply:
Unfortunately, jwt library need at least 1 character for verify function's second parameter(APP_SECRET)